setup_env.sh

```diff
@@ -24,4 +31,35 @@ if [ ! -f pathcheck-mobile-resources/environment/$HA/.env.bt.release ]; then
   exit 1
 fi
 
-cp -r pathcheck-mobile-resources/environment/$HA/. ./
\ No newline at end of file
+cp -r pathcheck-mobile-resources/environment/$HA/. ./
+
+# In order to read properly spaces on environment variables
+IFS='
+'
+
+if [ "$2" = "--renaming" ]; then
+# Loading the .env.bt
+export $(cat .env.bt | xargs)
```

Comment:
in general looks great, but this will need to handle `.env.release` too, no?

Reply:
Good point, I think we should be good by default to the `release` one unless the test flight and beta releases need a specific name.